data/reporter.htm

```diff
@@ -22,7 +22,7 @@
     </div>
 
     <div id="buttons">
-        <span id="appE10sEnabled"></span>
+        <span class="app-e10s-enabled"></span>
```

Comment:
haha did I? Damn. I usually mean JS variables but not class names. I guess camelCase is consistent though, I'll fix it 👍